src/helpers/validator.js

```diff
@@ -0,0 +1,41 @@
+import * as Yup from "yup";
+
+export const schema = Yup.object().shape({
+  nume: Yup.string().required("required"),
+  adresa_localitate: Yup.string().required("required"),
+  adresa_judet: Yup.string().required("required"),
+  adresa_strada: Yup.string().required("required"),
+  // adresa_numar: "",
+  // adresa_bloc: "",
+  // adresa_etaj: "",
+  // adresa_apartament: "",
+  cnp: Yup.string().required("required"),
+  ci_seria: Yup.string().required("required"),
+  // ci_numar: "",
+  // domiciliu_localitate: "",
+  // domiciliu_judet: "",
```

Comment:
We can remove those comments as well, at least for now.

Reply:
removed